snapshot/snapshot.go

```diff
@@ -343,14 +338,11 @@ func (o *snapshotter) Mounts(ctx context.Context, key string) ([]mount.Mount, er
 	return o.mounts(ctx, info.Labels, *snap)
 }
 
-func (o *snapshotter) prepareRemoteSnapshot(id string, labels map[string]string) error {
-	return o.fs.Mount(id, labels)
-}
-
 func (o *snapshotter) Prepare(ctx context.Context, key, parent string, opts ...snapshots.Opt) ([]mount.Mount, error) {
-	// `timer` can't be nil
-	timer := collector.NewSnapshotMetricsTimer(collector.SnapshotMethodPrepare)
-	defer timer.ObserveDuration()
+	log.L.Debugf("[Prepare] snapshot with key %s, parent %s", key, parent)
+	if timer := collector.NewSnapshotMetricsTimer(collector.SnapshotMethodRemove); timer != nil {
```

Comment:
Why using `SnapshotMethodRemove` instead of `collector.SnapshotMethodPrepare` here?

Reply:
My mistake, fixed.